### PR TITLE
Update Release-Notes.md

### PR DESCRIPTION
**Pull request by rich-howell**
Created release notes for the next release 23.01.xx

### CODE REVIEW

File: docs/Release-Notes/Release-Notes.md [hunk 1]
```diff
@@ -22,6 +22,35 @@ The release has notable of improvements of:
 
 A full accounting of the changes are listed below.  Thank you to everyone who contributed!
 
+### Actual 
+Version: 23.01.xx
```
Comment (j-f1) [inline docs/Release-Notes/Release-Notes.md:26]:
```suggestion
Version: 23.1.xx
```

Comment (rich-howell) [reply docs/Release-Notes/Release-Notes.md]: Fixed, sorry

File: docs/Release-Notes/Release-Notes.md [hunk 2]
```diff
@@ -2,17 +2,53 @@
...
+* [#440](https://github.com/actualbudget/actual/pull/440) Further iteration on the sidebar design -- thanks [j-f1](https://github.com/j-f1) 
+* [#441](https://github.com/actualbudget/actual/pull/441) Fix overlap of version info and server URL -- thanks [trevdor](https://github.com/trevdor)
+* [#442](https://github.com/actualbudget/actual/pull/442) 🔥 removal of react-native mobile apps -- thanks [MatissJanis](https://github.com/MatissJanis)
+* [#443](https://github.com/actualbudget/actual/pull/443) ⬆️ upgrade prettier and fix new issues -- thanks [MatissJanis](https://github.com/MatissJanis)
+
+### Actual Server 
+Version: 23.1.xx
+
+No pull requests were merged in this release.
+
```
Comment (shall0pass) [inline docs/Release-Notes/Release-Notes.md:51]: We lost the header for 22.12.03.

File: docs/Release-Notes/Release-Notes.md [hunk 3]
```diff
@@ -2,17 +2,53 @@
...
+* [#435](https://github.com/actualbudget/actual/pull/435) Read-only Responsive view -- thanks [trevdor](https://github.com/trevdor) 
+* [#437](https://github.com/actualbudget/actual/pull/437) chore: remove unsed vars & cleanups -- thanks [MatissJanis](https://github.com/MatissJanis) 
+* [#439](https://github.com/actualbudget/actual/pull/439) docs: add netlify as sponsors to README -- thanks [MatissJanis](https://github.com/MatissJanis) 
+* [#440](https://github.com/actualbudget/actual/pull/440) Further iteration on the sidebar design -- thanks [j-f1](https://github.com/j-f1) 
+* [#441](https://github.com/actualbudget/actual/pull/441) Fix overlap of version info and server URL -- thanks [trevdor](https://github.com/trevdor)
+* [#442](https://github.com/actualbudget/actual/pull/442) 🔥 removal of react-native mobile apps -- thanks [MatissJanis](https://github.com/MatissJanis)
+* [#443](https://github.com/actualbudget/actual/pull/443) ⬆️ upgrade prettier and fix new issues -- thanks [MatissJanis](https://github.com/MatissJanis)
+
+### Actual Server 
+Version: 23.1.xx
```
Comment (shall0pass) [inline docs/Release-Notes/Release-Notes.md:48]: Unless actual-server gets a version bump, the server version will still be reported as 22.12.09 in the app.

Comment (rich-howell) [reply docs/Release-Notes/Release-Notes.md:48]: It will be getting a version bump

File: docs/Release-Notes/Release-Notes.md (hunk 2 above)
Comment (rich-howell) [reply docs/Release-Notes/Release-Notes.md:48]: I totally overlooked how the document was arranged, this should be fixed.

**Review by MatissJanis — APPROVED** [review]

**Pull request merged**